entity-service/src/test/kotlin/com/egm/stellio/entity/repository/Neo4jRepositoryTests.kt

```diff
@@ -1149,6 +1150,185 @@ class Neo4jRepositoryTests : WithNeo4jContainer {
         )
     }
 
+    @Test
+    fun `it should retrieve attribute list`() {
+        val firstEntity = createEntity(
+            "urn:ngsi-ld:Beehive:TESTC".toUri(),
+            listOf("https://ontology.eglobalmark.com/apic#Beehive"),
+            mutableListOf(
+                Property(name = "temperature", value = 36),
+                Property(name = "humidity", value = 65)
+            )
+        )
+        val secondEntity = createEntity(
+            "urn:ngsi-ld:Sensor:TESTC".toUri(),
+            listOf("https://ontology.eglobalmark.com/apic#Sensor"),
+            mutableListOf(
+                Property(name = "deviceParameter", value = 30),
+                Property(name = "isContainedIn", value = 61)
+            ),
+            NgsiLdGeoPropertyInstance.toWktFormat(GeoPropertyType.Point, listOf(24.30623, 60.07966))
+        )
+
+        createRelationship(EntitySubjectNode(firstEntity.id), "observedBy", secondEntity.id)
+
+        val attributeName = neo4jRepository.getAttribute()
+        val expectedValue = listOf(
+            "deviceParameter",
+            "https://uri.etsi.org/ngsi-ld/location",
```

Comment:
it seems strange that `location` is not returned compacted (since it is a core NGSI-LD property)

Reply:
even in the previous existing tests of entity type its not returned as compacted form.